src/App/Fossa/Analyze/ScanSummary.hs

```diff
@@ -113,6 +119,14 @@ instance Pretty ScanCount where
       , pretty numWarnings <> " analysis " <> plural "warning" "warnings" numWarnings
       ]
 
+staticOnlyAnalysisMessage :: Doc a
+staticOnlyAnalysisMessage =
+  vcat
+    [ "Performed static-only analysis!!!"
```

Comment:
I appreciate the enthusiasm, but it reads very oddly in the otherwise quite dry FOSSA CLI output 😆
```suggestion
    [ "FOSSA CLI was constrained to static-only analysis."
```